src/script/backport-create-issue

```diff
@@ -159,11 +159,16 @@ def update_relations(r, issue, dry_run):
                 "tracker")
             continue
         if relation['relation_type'] != 'copied_to':
-            logging.error(self.url(issue) + " unexpected relation '" +
-                relation['relation_type'] + "' to " + self.url(other))
+            logging.error(url(issue) + " unexpected relation '" +
+                relation['relation_type'] + "' to " + url(other))
             success = False
             continue
         release = get_release(other)
+        if release in existing_backports:
+            logging.error(url(issue) + " duplicate " + release +
```

Comment:
could add a " - " (or some separator) between the issue and the corresponding error message.

Reply:
Probably a good idea, but over the years I've grown used to how the script communicates with me, so I'm reluctant to change that! Heh.